3-reveal/migrations/6-MDA/Zambia-2020/deploy/mda_structures.psql

```diff
@@ -0,0 +1,101 @@
+-- Deploy reveal_mda_zambia_2020:mda_structures to pg
+-- requires: reveal_transaction_tables:plans
+-- requires: reveal_transaction_tables:events
+-- requires: reveal_transaction_tables:jurisdictions
+-- requires: reveal_transaction_tables:clients
+-- requires: reveal_transaction_tables:locations
+-- requires: reveal_transaction_tables:tasks
+-- requires: reveal_database_views:jurisdictions_materialized_view
+
+BEGIN;
+
+SET search_path TO :"schema",public;
+
+CREATE OR REPLACE VIEW ntd_dispense_structures AS
```

Comment:
@baokiddy I changed this view to be based on structures as opposed to tasks.  It turned out to be easier and led to a less expensive query.